dev/com.ibm.ws.security.oauth/resources/com/ibm/ws/security/oauth20/internal/resources/OAuthMessages.nlsprops

```diff
@@ -141,16 +141,16 @@ OAUTH_CLIENT_REGISTRATION_CLIENTID_EXISTS=CWWKS1429E: Client id {0} already exis
 OAUTH_CLIENT_REGISTRATION_CLIENTID_EXISTS.explanation=The operation failed as the client id already exists.  
 OAUTH_CLIENT_REGISTRATION_CLIENTID_EXISTS.useraction=Review the API documentation for this service, and retry the request with the appropriate parameters.
 
-OAUTH_CLIENT_REGISTRATION_CLIENT_SECRET_UPDATE_FAILURE=CWWKS1430E: An update of the client fails.
-OAUTH_CLIENT_REGISTRATION_CLIENT_SECRET_UPDATE_FAILURE.explanation=The update request indicates that the client secret be preserved, but the existing registration has no secret set.  
+OAUTH_CLIENT_REGISTRATION_CLIENT_SECRET_UPDATE_FAILURE=CWWKS1430E: An update of the client fails. The update request indicates that the client secret should be preserved, but the existing registration has no secret set.
```

Comment:
Change
'An update of the client fails. The update request indicates that the client secret should be preserved, but the existing registration has no secret set.'
to
'An update of the client fails. The update request indicates that the client secret must be preserved but the existing registration has no secret set.'